hardware/raspberrypi/bootmodes/bootflow_2711.md

```diff
@@ -1,31 +1,67 @@
-# Pi4 Bootflow
+# Raspberry Pi4, Pi400 and CM4 bootflow
 
-The Pi4 with the BCM2711 SoC has a new, more sophisticated boot process. The addition of an EEPROM means that the `bootcode.bin` file found in `/boot` is no longer required. Details of the EEPROM can be found [here](../booteeprom.md).
+This page describes the bootflow for BCM2711 based products. The main difference betweeen this and previous products is that the second stage bootloader (bootcode.bin) is loaded from an SPI flash [EEPROM](../booteeprom.md). 
```

Comment:
Dunno if it's worth clarifying "(bootcode.bin)" to "(bootcode.bin on previous models)"?